sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobClient.java

```diff
@@ -629,8 +639,8 @@ public VoidResponse setMetadata(Metadata metadata, BlobAccessConditions accessCo
      * @param tier The new tier for the blob.
      * @return A response containing status code and HTTP headers.
      */
-    public VoidResponse setTier(AccessTier tier) {
-        return this.setTier(tier, null, null);
+    public Void setTier(AccessTier tier) {
```

Comment:
This should be `void`, correct?

Reply:
If so, please apply in all applicable cases.

Reply:
To make this `void` we will have to call the `async` equivalent of this method and not the overload as we have been doing everywhere else.
Something like this,
```java
public void setTier(AccessTier tier) {
        blobAsyncClient.setTierWithResponse(tier, null, Context.NONE).block();
        return;
    }
```
still do it ?

Reply:
Could you just not return anything with the code as it is today?

Reply:
updated.